development/tests/test_views_forms.py

```diff
@@ -1,6 +1,6 @@
 from auth_app.models import User, Bot
 from django.test import RequestFactory, TestCase
-from ..views import ChallengeView, MatchListView, MyBotsView
+from ..views import ChallengeView, MatchListView, MyBotsView, AddBotView
```

Comment:
nit: remember that multiples import must be in multiple lines, not in one line. Besides, they have to be in alphabetic order.
```
from ..views import (
   AddBotView,
   ChallengeView,
   MatchListView,
   MyBotsView,
)
```

Reply:
done